TUnit.Analyzers.Tests/MethodDataSourceAnalyzerTests.cs

```diff
@@ -429,7 +429,31 @@ public static int Data(bool flag)
                                 }
                             }
                             """;
-        
+
+        await Verifier.VerifyAnalyzerAsync(text).ConfigureAwait(false);
```

Comment:
@thomhurst How would you feel about changing this code pattern to:
```cs
public async Task Test()
{
  await Verifier
    .VerifyAnalyzerAsync(
      """
      // cs code
      """
    )
    .ConfigureAwait(false);
```

This would allow using `[StringSyntax("c#-test")]`, which would add syntax highlighting to these strings.

The alternative is a bit more manual:
```cs
const string text = """ // lang=c#-test
...
```

Reply:
I didn't even know about that... That sounds like it could be nice!

Reply:
I'll look at doing that next week - I should have some free time.